Documentation/Signoffs/CommunicationsSignoff.md

```diff
@@ -0,0 +1,38 @@
+# Communication Signoff
+## FUNCTION OF THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The Communication subsystem is responsible for receiving data from the Microcontroller subsystem, connecting the device to the Internet, and formatting and sending the received data to the Web & Wireless subsystem.  This subsystem will utilize 4G LTE (Long-term Evolution) as a standard in telecommunications to connect to the Internet for its near global coverage.  
+
+## CONSTRAINTS ON THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  The constraints given below are generalized to allow for comparison of any potential contender for the solution.  Constraints on the subsystem are as follows:
+1. The component must be able to utilize 4G LTE telecommunication.  4G LTE operating frequencies include 600 MHz, 700 MHz, 1700/2100 MHz, 2300 MHz, and 2500 MHz bands.  This is the chosen method of accessing the Internet due to its coverage and support in place.
+2. The component must have a slot and support for a Subscriber Identity Module card (SIM).  The component must be able to use this SIM card and its identifying information to access the specified broadband cellular network related to the SIM card's service provider.
```

Comment:
Reword to just say it must support SIM card which is required for connection to cellular network.

Should eSIM be considered?

Reply:
Reworded.  eSIM was not considered, but it would work as it accomplished the same goal.  There would still need to special storage for the identifying data.